### DISCUSSION TITLE
How to I change the repository my githup will push to?

### DISCUSSION THREAD

**Discussion by strikeouts27** [community/community#65001] [Programming Help]
### Body

I ordered a push and it pushed my code to the wrong repostiory. I checked stack overflow and it tells me how to see it but not how to change the desitination? 

I believe I will have to try to fix my other repository once this is over. sigh.

It keeps trying to send code to: 

(base) andrewstribling@Andrews-MBP news % git remote -v 
origin  git@github.com:strikeouts27/news.git (fetch)
origin  git@github.com:strikeouts27/news.git (push)
(base) andrewstribling@Andrews-MBP news % 

I do not want it to go to strikeouts27/news.git. when I try to look at my github page that repository does not show.
it won't let me create a new repository and synchronize either. When I try to create a new repository the pushed goes to this git@github account. I don't understand why this repository will not show up on my github account.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arafajs** [comment]
To change the repository that your GitHub will push to, you need to update the remote configuration in your local Git repository.

The remote configuration tells Git where to find the remote repository. In your case, the remote repository is currently set to `strikeouts27/news.git.`

To change the remote repository, you can use the git remote set-url command. The syntax for this command is:

`git remote set-url <remote_name> <new_url>`
In your case, the command would be:

`git remote set-url origin <your_github_username>/<new_repository_name>.git`
For example, if your GitHub username is andrewstribbing and you want to change the repository to my-new-repository, the command would be:

`git remote set-url origin andrewstribling/my-new-repository.git`
Once you have updated the remote configuration, you can push your changes to the new repository by running the git push command.

To verify that the remote repository has been changed, you can run the git remote -v command again. The output of this command should show the new remote repository URL.

Here are the steps in detail:

1. Open a terminal window and navigate to the directory that contains your local Git repository.
2. Run the git remote -v command to see the current remote configuration.
3. Update the remote configuration by running the git remote set-url command.
4. Run the git push command to push your changes to the new remote repository.
5. Run the git remote -v command again to verify that the remote configuration has been changed.

**Reply by strikeouts27** [reply]
I have tried to type in:

git remote set-url origin strikeouts27?django_news.git 

it seems to accept the command and nothing happens.

I run git add . 
and than I run git commit -m "testing push"

Still running into problems. Removing all associated repositories and will try to create a new one.

**Reply by strikeouts27** [reply]
even when I delete all of the repositories and try to have one folder it still refuses me! 

Output: 

(base) andrewstribling@Andrews-MBP news % echo "# django_news" >> README.md
git init
git add README.md
git commit -m "first commit"
git branch -M main
git remote add origin https://github.com/strikeouts27/django_news.git
git push -u origin main
Reinitialized existing Git repository in /Users/andrewstribling/Desktop/Programming/projects/django/news/.git/
On branch main
Changes not staged for commit:
       modified:   README.MD

no changes added to commit
fatal: remote origin already exists.
fatal: 'strikeouts27/django_news.git' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
